### PR TITLE
Hardening/test for metrics

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan

### CODE REVIEW

File: lib/models/emailAction.js [hunk 1]
```diff
@@ -52,8 +52,13 @@ function SendMail(action, event, callback) {
 
         transporter.sendMail(mailOptions, function(err, info) {
             logger.debug('emailAction.SendMail %j %j %j', mailOptions, err, info);
+            // Not a HTTP request, must count outgoingTransaction
```
Comment (fgalan) [inline lib/models/emailAction.js]: As discussed by skype, this (and the other two similar comments) could use:

// Not a HTTP request, so outgoingTransacion hasn't already counted and must be counted now

Comment (fgalan) [inline lib/models/emailAction.js]: NTC, aready done

(I was looking to an older version of the code when I wrote the comment, sorry...)

Comment (fgalan) [inline lib/models/emailAction.js]: Mmmm... not fully

It seems 2/3 occurence were changed, but not the one on which this comment has been created...

Comment (crbrox) [inline lib/models/emailAction.js]: fixed c54e53e9296ad34a177121734466e6507636beb0

**Comment by fgalan** [comment]
LGTM

**Pull request merged**